src/test_reference.jl

```diff
@@ -1,37 +1,85 @@
 """
-    @test_reference filename ex [kw...]
+    @test_reference filename ex [test] [render]
```

Comment:
Not really changed by this PR
but we should change `ex` to `expr`
since `ex` couild be `exception`, or `example`